libraries/Microsoft.Bot.Builder.Dialogs/DialogManager.cs

```diff
@@ -116,13 +116,24 @@ public Dialog RootDialog
         /// </value>
         public int? ExpireAfter { get; set; }
 
+        /// <summary>
+        /// IBot on turn method.
+        /// </summary>
+        /// <param name="turnContext">turn context.</param>
+        /// <param name="cancellationToken">cancellationToken.</param>
+        /// <returns>task.</returns>
+        Task IBot.OnTurnAsync(ITurnContext turnContext, CancellationToken cancellationToken)
```

Comment:
This is redundant and can be removed, DialogManager is an IBot and it implicitly implements OnTurnAsync below, there is no need for an explicit interface implementation here, and IBot will be able to access the OnTurnAsync method defined below.